spec/lib/iiif_manifest/v3/manifest_factory_spec.rb

```diff
@@ -147,10 +153,24 @@ def display_content
         expect(result['items'].first['items'].first['items'].first['target']).to eq result['items'].first['id']
         expect(result['items'].first['items'].first['items'].first['body']['type']).to eq 'Image'
         expect(result['items'].first['items'].first['items'].first['body']['id']).not_to be_empty
-        expect(result['items'].first['items'].first['items'].first['body']['height']).to eq 100
-        expect(result['items'].first['items'].first['items'].first['body']['width']).to eq 100
+        expect(result['items'].first['items'].first['items'].first['body']['height']).to eq 1500
+        expect(result['items'].first['items'].first['items'].first['body']['width']).to eq 2000
         expect(result['items'].first['items'].first['items'].first['body']['format']).to eq 'image/jpeg'
       end
+
+      it 'has a thumbnail property' do
+        allow(book_presenter).to receive(:member_presenters).and_return([file_presenter])
+
+        result
```

Comment:
What's this line doing?

Reply:
This was following other tests that had it so I thought it was needed.  Looks like it is so I can remove from here and other places as well.